google/cloud/internal/log_wrapper_test.cc

```diff
@@ -317,7 +318,9 @@ TEST(LogWrapper, StatusValueError) {
   grpc::ClientContext context;
   btproto::MutateRowRequest request;
   btproto::MutateRowResponse response;
-  LogWrapper(mock, &context, request, &response, "in-test", {});
+  auto const r = LogWrapper(mock, &context, request, &response, "in-test", {});
+  EXPECT_EQ(r.error_code(), status.error_code());
```

Comment:
I would use `EXPECT_EQ(status, r);` in this case.  Also consider `StatusIs()` which may produce better error messages.

Reply:
The external `grpc::Status` doesn't have `op==`, and `StatusIs` doesn't work with `grpc::Status`, which has differently named methods like `error_message()` instead of `message(). I tried both of those already :smile:

Reply:
Doh!